packages/fxa-shared/db/models/auth/refresh-token-meta-data.ts

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export class RefreshTokenMetadata {
+  public lastUsedAt: Date;
+
+  constructor(lastUsedAt?: Date | null) {
+    this.lastUsedAt = lastUsedAt || new Date();
+  }
+
+  /**
+   *
```

Comment:
nit: In general, I've always found function comments like this to be a lil strange without a description. I would probably remove comment or add a blurb about it.